client/src/components/auth/authComponents/RegistrationForm.jsx

```diff
@@ -64,7 +64,7 @@ const RegistrationForm = () => {
       .required('Please enter your password')
       .matches(
         /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/,
-        'Password is not strong enough',
+        'Password must contain 8 characters, 1 uppercase, \n1 lowercase, 1 number and 1 symbol.',
```

Comment:
it should say must contain at least 8 characters?

Reply:
also, should the hover password hint still be there if we have this then?

Reply:
yep, changed to 'at least'.
I think the hover password hint will still be needed cause that's the only way to know the password restriction before pressing sign up button.